.yarnrc.yml

```diff
@@ -0,0 +1,19 @@
+# We use allow-scripts to manually enable scripts and then run them
```

Comment:
I've found that this file will get regenerated if you add a new plugin. So that's one thing to know about this file :(

Reply:
I had this problem as well. I was preserving them manually. Pretty inconvenient.